python/artm/regularizers.py

```diff
@@ -66,29 +67,63 @@ def _update_config(self, obj, first=False):
 
 
 class Regularizers(object):
-    def __init__(self, master):
+    def __init__(self, master, force_add=False):
```

Comment:
It sounds reasonably.
BTW, is it good idea add key `overwite=True` instead of implementing `add_or_replace` method?
I.e. what if we have the following convention:
* `model.regularizers['name'] = SomeReg()` and `model.regularizers.add(..., overwrite=True)` may overwrite previous value.
* `model.regularizers.add(...)` may not ovewrite prevous value, instead it raises exceptions with some useful messages like "... if you want to do it please use overwrite=True".